code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -1,5 +1,13 @@
 #define ELZUOSE_EMAG_COLORS list("#00ffff", "#ffc0cb", "#9400D3", "#4B0082", "#0000FF", "#00FF00", "#FFFF00", "#FF7F00", "#FF0000")
 
+#define GOOD_SOIL list(/turf/open/floor/plating/grass, /turf/open/floor/plating/dirt, /turf/open/floor/ship/dirt, /turf/open/floor/grass/ship, /turf/open/floor/plating/asteroid/whitesands/grass, /turf/open/floor/grass/fairy/beach)
+
+#define DIG_TIME (7.5 SECONDS)
+
+#define ROOT_TIME (3 SECONDS)
+
+#define ROOT_CHARGE_GAIN (5 * ELZUOSE_CHARGE_SCALING_MULTIPLIER)
```

Comment:
No need to space these apart, looks messy. `#undef` after you're done using it if you don't plan on using it in another file.